embulk-core/src/main/java/org/embulk/spi/time/TimestampSerDe.java

```diff
@@ -9,66 +9,41 @@
 import com.fasterxml.jackson.databind.module.SimpleModule;
 import com.fasterxml.jackson.module.guice.ObjectMapperModule;
 import java.io.IOException;
-import java.time.LocalDateTime;
-import java.time.ZoneOffset;
-import java.time.format.DateTimeFormatter;
-import java.util.regex.Matcher;
-import java.util.regex.Pattern;
 
 public class TimestampSerDe {
     public static void configure(ObjectMapperModule mapper) {
         SimpleModule module = new SimpleModule();
-        module.addSerializer(Timestamp.class, new DateTimeZoneSerializer());
-        module.addDeserializer(Timestamp.class, new DateTimeZoneDeserializer());
+        module.addSerializer(Timestamp.class, new TimestampSerializer());
+        module.addDeserializer(Timestamp.class, new TimestampDeserializer());
         mapper.registerModule(module);
     }
 
-    public static class DateTimeZoneSerializer extends JsonSerializer<Timestamp> {
```

Comment:
Historical mistake. `TimestampSerDe` has had `DateTimeZone(De)Serializer` in it...

Reply:
I see...